packages/slate-react/src/utils/scroll-to-selection.js

```diff
@@ -11,23 +33,45 @@ import isBackward from 'selection-is-backward'
 function scrollToSelection(selection) {
   if (!selection.anchorNode) return
 
-  const window = getWindow(selection.anchorNode)
+  const scrollContainer = findScrollContainer(selection.anchorNode)
+  const scrollContainerIsWindow = scrollContainer == scrollContainer.window
```

Comment:
Will `scrollContainer == scrollContainer.window` ever be true? I'm not sure what the `.window` property is?